src/reducers/conversationReducer.js

```diff
@@ -276,7 +280,35 @@ export const markAsRead = (state, action) => {
     }
   })
 
-  state.list = finalList;
+  state.conversationlist = finalList;
+}
+
+export const conversationBanListSuccess = (state, action) => {
+  state.banList = action.payload;
+}
+
+export const banConversationUserSuccess = (state, action) => {
+  const { conversation, userIds, displayName } = action.payload;
+
+  const banUserList = userIds.map(userId => {
+    return {userId, user: {displayName}};
+  });
+
+  const finalList = [...state.banList, ...banUserList];
+  state.banList = finalList;
+}
+
+export const unbanConversationUserSuccess = (state, action) => {
+  const { conversation, userIds } = action.payload;
+
+  const finalList = [];
+  state.banList.forEach((user) => {
+    if (!userIds.includes(user.userId)) {
+      finalList.push(user);
+    }
+  });
```

Comment:
@pardeep-kumar-bs you cloud use here `slice` function.

Reply:
Use Filter

Reply:
Done.